### ISSUE TITLE
Enhancement: More appropriate semenatic markup could be added to How we work section

### ISSUE THREAD

**Issue by neilmispelaar** [DTS-STN/Alpha-Site#230]
### Description

The "How we work" section on the "about" page uses an ol (ordered list) to mark up the list. Immediately after the list item, a number is written in. (This is better described in the screenshot below where you can see two number ones, two number two's, two number three's, etc).

One issue is the "stutter effect - screen readers will read out the numbers twice: "List item with four elements. Element number one. One. Displaying our early works in progress. List item number two. Two. Encouraging people of diverse backgrounds....". Not the end of the world but not a perfect experience for screen readers.

The second issue is that ol (ordered list) are reserved for lists of sequence. These are generally used for recipes or instructions, where you have to do step 1, before going on to step 2, and so on and so forth. How the page reads (to me anyway), is that this is more of a general list of things that we do, and we don't do these things in sequence, or rather the sequence of events doesn't matter. We may blog before we use new technology.

### How to fix

Changing the list to an ul (unordered list where the sequence doesn't have meaning) and using an aria role=presentation on just the numbers in the list item could enhance the semantic markup.

### WCAG Compliance 

https://www.w3.org/WAI/WCAG21/Techniques/html/H48

### Screenshot

<img width="1700" alt="Screen Shot 2021-07-21 at 1 12 39 PM" src="https://user-images.githubusercontent.com/9325038/126535264-6683655c-04fc-48f9-a35b-66d0029f5f17.png">
